src/Pokemon.java

```diff
@@ -0,0 +1,69 @@
+public abstract class Pokemon {
+    private int level;
+    private int hp;
+    private String food;
+    private String sound;
+    private String name;
+    private String type;
+
+
+
+    //constructor
```

Comment:
Deze comment is overbodig, ik kan zo wel zien dat het een constructor is (dat geld ook voor setter en getter comments